### ISSUE TITLE
Problems from 2.7.0

### ISSUE THREAD

**Issue by frederikhors** [sveltejs/eslint-plugin-svelte3#32]
From `"eslint-plugin-svelte3": "2.7.0",` both my IDE with eslint support and `npx eslint src/**` from cli complains about:

`3:3   error  Import in body of module; reorder to top  import/first`

which is disable like this in `.eslintrc.js`:

```js
module.exports = {
  root: true,
  parserOptions: {
    ecmaVersion: 2019,
    sourceType: 'module'
  },
  env: {
    es6: true,
    browser: true,
    node: true
  },
  extends: [
    'eslint:recommended',
    'standard'
  ],
  plugins: ['svelte3'],
  overrides: [
    {
      files: ['*.svelte'],
      processor: 'svelte3/svelte3',
      rules: {
        'import/first': 0,
        'import/no-duplicates': 0,
        'import/no-mutable-exports': 0
      }
    }
  ]
}
```

With `2.6.0` everything ok.

Why?

**Comment by Conduitry** [comment]
Sorry about that! I suppose the addition of [named code blocks](https://github.com/sveltejs/eslint-plugin-svelte3#named-code-blocks) was sort of a breaking change. You need to disable the rules on `'*.svelte/*.js'` now. I'll explain this better in the docs soon.

**Comment by Conduitry** [comment]
Actually, I think I'll release a bugfix version that puts the named code blocks feature behind a setting so you have to explicitly opt in. I really wasn't thinking about how much of a breaking change this would be.

**Comment by Conduitry** [comment]
2.7.1 has been released. Does this fix the errors for you?

**Comment by frederikhors** [comment]
> 2.7.1 has been released. Does this fix the errors for you?

yep.

**Comment by nickelghost** [comment]
I have the same issue, but only with some files.
![Screenshot from 2020-01-22 07-13-14](https://user-images.githubusercontent.com/16365643/72869892-e4621800-3ce6-11ea-9d15-3f02cc021e17.png)

**Comment by Conduitry** [comment]
See <https://github.com/sveltejs/eslint-plugin-svelte3/blob/master/OTHER_PLUGINS.md#eslint-plugin-import>

**Comment by mckravchyk** [comment]
@nickelghost It seems like it does not work if you are using a `$: ` declaration. It also does not work when there are multiple script tags and importing something that is not in the first script tag.
